Content.Shared/Xenoarchaeology/Artifact/XAT/Components/XATTimerComponent.cs

```diff
@@ -0,0 +1,16 @@
+using Robust.Shared.GameStates;
+
+namespace Content.Shared.Xenoarchaeology.Artifact.XAT.Components;
+
+/// <summary>
+/// This is used for a xenoarch trigger that self-activates at a regular interval
+/// </summary>
+[RegisterComponent, NetworkedComponent, Access(typeof(XATTimerSystem)), AutoGenerateComponentState, AutoGenerateComponentPause]
+public sealed partial class XATTimerComponent : Component
```

Comment:
need to re-check if that is working with examine. Need to re-check usages

Reply:
Examine is of no real consequence there but it is used and it is working. Wierd shit if you ask me, not really a good idea but it creates some difference with this delay amount.